src/map.cpp

```diff
@@ -2726,22 +2726,41 @@ void map::drop_items( const tripoint &p )
     // rather than disappearing if it would be overloaded
 
     tripoint below( p );
+    int height_fallen = 0;
     while( !has_floor( below ) ) {
         below.z--;
+        height_fallen++;
     }
 
     if( below == p ) {
         return;
     }
 
-    for( const item &i : items ) {
-        // TODO: Bash the item up before adding it
-        // TODO: Bash the creature, terrain, furniture and vehicles on the tile
+    float damage = 0.0f;
+    for( item &i : items ) {
+        units::mass wt_dropped = i.weight();
+        damage += 10 * to_kilogram( wt_dropped ) * sqrt( height_fallen );
```

Comment:
Flat increase of damage over sum of weights doesn't make sense, this effectively grants armor penetration properties and will do more damage when selecting multiple items to drop than if you drop one at a time, unless monster has 0 bash resist

This also doesn't account for density(surface area for air resistance), collision type/momentum change over time etc

If 40 500g pillows do the same damage as 20kg battery, or 20kg mattress does the same damage as 20kg battery this seems more of a looney tunes meme than a game mechanic

Reply:
This will not grant additional damage if dropped 1 at a time because damage is always 10*h^1/2*(x1 + x2 ... )

And armor penetration makes sense because this is bash damage like dropping a tank on top of an army helmet

Hmm perhaps air resistance is something that can be looked into, not sure if this is something we should add as it might cause all sorts of game aspects to go out of wack such as bow arrow movement, throwing objects, vehicle movement etc.

Also I think 40 500g pillows should do same damage as a 20kg battery if we ignore the air resistance

Reply:
Repeat that to yourself,  "40 500g pillows should deal the same damage as a 20kg battery"
That is absolutely not correct.
At falling speeds,  hardness and density are critical components of damage dealt,  this must be represented for this to be mergeable.

Reply:
I could add a bulk modulus component(how far the item travels after hitting another to come to a stop) to each item which could account for this but density definitely should not effect the damage.

Reply:
Actually looking into this, since density is related to bulk modulus, using density to make this calculation instead